pontoon/base/templatetags/helpers.py

```diff
@@ -1,8 +1,9 @@
 import cgi
 import datetime
 import json
-import urllib
-import urlparse
+
+from six import text_type
+from six.moves.urllib import parse as urlparse
```

Comment:
I find this a bit worrying, because there is a package that is called ``urlparse``. We might end up with people thinking it's the actual library and try to do things with it that are not supported by ``six.moves.urllib.parse``. Can you name it something different that conveys the fact that it's from the ``six`` library?